tests/system/small/test_session.py

```diff
@@ -353,6 +355,45 @@ def test_read_gbq_table_wildcard_with_filter(session: bigframes.Session):
     assert df.shape == (348485, 32)
 
 
+@pytest.mark.parametrize(
+    ("config"),
+    [
+        {
+            "query": {
+                "useQueryCache": True,
+                "maximumBytesBilled": "1000000000",
+                "timeoutMs": 10000,
+            }
+        },
+        pytest.param(
+            {"query": {"useQueryCache": True, "timeoutMs": 50}},
+            marks=pytest.mark.xfail(
+                raises=google.api_core.exceptions.BadRequest,
```

Comment:
curious why timeout 50ms is a bad request?

Reply:
It's an error that the time set is too short, this is a quick way to prove that the parameter works.

Reply:
Makes sense, thanks!